### PR TITLE
add github release event

### PR DESCRIPTION
**Pull request by Pantani**
### What does this MR does?

Add Github release event support

### Why we need it?

For we can track new release events from the GitHub repo.

### How to test?

- Install the bot
- subscribe to a new release
- wait the release events: `!github subscribe <REPO> releases`

### CODE REVIEW

**Comment by Pantani** [comment]
@joshblum @marceloneil @mmaxim @songgao

File: base/git/git.go [hunk 1]
```diff
@@ -130,3 +130,32 @@ func FormatPullRequestMsg(provider int, action string, username string, repo str
...
+		res = fmt.Sprintf("%s published the relase %s (%s) on %s: \n%s\n", username, version, name, repo, changes)
+		res += releaseUrl
+	case "create", "created":
+		res = fmt.Sprintf("%s created the relase %s (%s) on %s: \n%s\n", username, version, name, repo, changes)
+		res += releaseUrl
+	case "edit", "edited":
+		res = fmt.Sprintf("%s edited the relase %s (%s) on %s: \n%s\n", username, version, name, repo, changes)
+		res += releaseUrl
+	case "delete", "deleted":
+		res = fmt.Sprintf("%s deleted the relase %s (%s) on %s: \n%s\n", username, version, name, repo, changes)
```
Comment (marceloneil) [inline base/git/git.go:157]: Typo here, should be "release" instead of "relase"

Comment (Pantani) [reply base/git/git.go]: solved, thanks!!!

**Comment by Pantani** [comment]
@marceloneil, can you help me? I can't request the reviewers. I think it's done

**Comment by joshblum** [comment]
This looks great @Pantani, thanks!

**Comment by joshblum** [comment]
cc @hyperobject

**Pull request merged**